test/TestImageFitting.cc

```diff
@@ -60,38 +102,34 @@ class ImageFittingTest : public ::testing::Test {
 
         std::string file_path =
             ImageGenerator::GeneratedFitsImagePath("128 128", fmt::format("--gaussian-model {} -s 0", gaussian_model_string));
-        std::shared_ptr<carta::FileLoader> loader(carta::FileLoader::GetLoader(file_path));
-        std::unique_ptr<TestFrame> frame(new TestFrame(0, loader, "0"));
-
-        CARTA::FittingResponse fitting_response;
-        std::unique_ptr<carta::ImageFitter> image_fitter(new carta::ImageFitter(frame->GetWidth(), frame->GetHeight()));
-        bool success = image_fitter->FitImage(frame->GetImageCacheData(), _initial_values, fitting_response);
+        return file_path;
+    }
 
+    void CompareResults(CARTA::FittingResponse fitting_response, bool success, std::string failed_message) {
```

Comment:
Use const references for fitting_response and string parameters

Reply:
This is fixed